docs/nightscout/remote-errors.md

```diff
@@ -77,71 +77,64 @@ The references to Caregiver below is the person sending the commands. There are
-
-Check these settings on the Looper’s device, not the Caregivers. Several of these are related to Apple suppressing notifications.
-
-* Notifications
-    * Settings -> Notifications ->&nbsp;_<span translate="no">Loop</span>_
-    * Turn on “Allow Notifications”
-    * Turn on “Time Sensitive Notifications”
-* Focus Modes
-    * For all focus modes (ex: Do Not Disturb, Sleep), make sure&nbsp;_<span translate="no">Loop</span>_&nbsp;is listed as an app allowing Notifications.
-    * To Adjust
-        * Settings -> Focus
-        * Select the focus mode (ex: Do Not Disturb, Sleep)
-        * Under “Allow Notifications, tap “Apps”
-        * Add “&nbsp;_<span translate="no">Loop</span>_&nbsp;” to the list.               
-        * Turn on “Time Sensitive Notifications”.
-* Background App Refresh
-    * Settings -> General -> Background App Refresh
-    * Select “On” up top
-    * Activate&nbsp;_<span translate="no">Loop</span>_&nbsp;toggle in list
-* Lower Power Mode
-    * Turn off if able
-
-Some other things to try on the Looper’s phone:
-
-* Reboot phone
-    * This sometimes resets Apple’s push notification limit.
-* Try wifi instead of cellular, if able
-    * Apple may not deliver notifications on cellular as often as wifi.
-* Charge the phone
-    * If the battery is low, iOS may not deliver notifications to save battery life.
-* Limit the number of&nbsp;_<span translate="no">Loop</span>_&nbsp;commands you send in a short period. Apple may throttle notifications if too many are received. (i.e. no more than 1 or 2 per hour may help).
-* Consider disabling “spammy” notifications from other apps. This is only a theory, but it is possible that other apps can cause the system to throttle all notifications, including&nbsp;_<span translate="no">Loop</span>_.
-* Wait 24 hours or so as it often just takes time for the push notification limits to “reset”.
-* iOS 15.3.x: Note there are reports of Remote notifications not being received to the Loopers device for iOS version 15.3 and 15.3.1. This is fixed in iOS version 15.4.
+Apple push notifications may not always reach your app due to various settings or Apple's intentional limitations. This issue can manifest in several ways:
+
+* The push notification banner never appears on the Looper’s device.
+* The push notification banner appears, but nothing happens in&nbsp;_<span translate="no">Loop</span>_&nbsp;(no error or success message afterward).
+* An error message in *Nightscout* or *Loop Caregiver* indicates the Password (OTP) is expired.
+
+While&nbsp;_<span translate="no">Loop</span>_&nbsp;cannot control the timely delivery of push notifications, there are several steps you can take to mitigate these issues. Note that rebuilding&nbsp;_<span translate="no">Loop</span>_&nbsp;or *Loop Caregiver* generally will not resolve the problem.
+
```

Comment:
Joined the 2 sections below into one list numbered 1 to 10. That should make it easier to review the "10 things" when they are getting expired command issues.